src/seismometer/seismogram.py

```diff
@@ -272,7 +272,7 @@ def _set_df_counts(self):
         if feature_counts > 0:
             self._feature_counts = feature_counts
         else:
-            self._feature_counts = f"~{max(0, len(self.dataframe.columns) - self.config.prediction_columns)}"
+            self._feature_counts = f"~{max(0, len(set(self.dataframe.columns) - set(self.config.prediction_columns)))}"
```

Comment:
Dumb question, if `feature_counts` is defined, we get a number 
if not, we get a string. Is the expected?

Your unit tests made me curious.

Reply:
Yes, the tilda is intentional to indicate an estimate since the lack of specification means this is potentially just an estimate